LinkBase/LinkBase/HudView.swift

```diff
@@ -7,3 +7,64 @@
 //
 
 import Foundation
+import UIKit
+
+class HudView: UIView{
+    var text = ""
+    
+    class func hud(inView view: UIView, animated: Bool) -> HudView {
+        
+        let hudView = HudView(frame: view.bounds)
+        hudView.isOpaque = false
+        
+        view.addSubview(hudView)
+        view.isUserInteractionEnabled = true
+        
+        hudView.show(animated: animated)
+        return hudView
+    }
+    
+    override func draw(_ rect: CGRect) {
+        
+        let boxWidth: CGFloat = 96
```

Comment:
for these constants, are they based off of auto layout settings? Im just worried that this will be different in different sizes 🤔